server/views/partials/hearingDetailsVariations/hearingTableExpansions/hearingTableCompleteDismissed.njk

```diff
@@ -1,18 +1,41 @@
+{% set adjudicatorChangeLink = {
+  items: [
+    {
+      href: adjudicationUrls.enterHearingOutcome.urls.edit(adjudicationNumber, hearingId), 
+      text: "Change",
+      visuallyHiddenText: ' name of adjudicator',
+      attributes: {"data-qa": "change-link-hearing-outcome-adjudicator-name" }
+    }
+  ]
+} %}
+{% set outcomeChangeLink = {
+  items: [
+    {
+      href: adjudicationUrls.enterHearingOutcome.urls.edit(adjudicationNumber, hearingId), 
+      text: "Change",
+      visuallyHiddenText: ' outcome',
+      attributes: {"data-qa": "change-link-hearing-outcome-outcome" }
+    }
+  ]
+} %}
+
```

Comment:
I tried having this section as a partial so we could just include it in the multiple files where it's required but I'm not sure that 'set' registers when you use it like that - I might try again at a later date to see if i can get it to work another way